CMakeLists.txt

```diff
@@ -231,3 +232,40 @@ endif()
 if (BUILD_TESTING)
         add_subdirectory(test)
 endif()
+
+################################################################################
+# Packaging
+################################################################################
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
+if(NOT BUILD_SHARED_LIBS)
+        set(CPACK_PACKAGE_NAME "${CPACK_PACKAGE_NAME}-static")
```

Comment:
Debian standard is to put the static variant of the library into the `-dev` package.  (Since a static library is never a runtime dependency, only a development dependency.)

Reply:
Is the standard to also only include the headers in the `-dev` package?

Reply:
Yes.  Meant to mention that too.  Thanks!